app/controllers/transactions_controller.rb

```diff
@@ -38,9 +38,21 @@ def index
     end
   end
 
-  def in_review_with_search
+  def in_review
     @recently_reviewed = current_user.administered_order_details.recently_reviewed.paginate(page: params[:page])
-    @order_details = current_user.administered_order_details.in_review
+    order_details = current_user.administered_order_details.in_review
+
+    @search_form = TransactionSearch::SearchForm.new(params[:search])
+    @search = TransactionSearch::Searcher.new(TransactionSearch::FacilitySearcher,
```

Comment:
Do we use this exact list elsewhere? I'm just wondering if it makes sense to wrap up some of the common filter lists like we have the default, so we could do something like `TransactionSearch::Searcher.new(:my_transactions)` or a `TransactionSearch::Searcher.my_transactions` class method and have some sort of mapping of keys to the the list? Just a thought; you probably have a better sense of where we're using custom lists. I wouldn't have a problem punting on it for now, either.

Reply:
I vote we punt for now, the degree of duplication isn't large enough to make this a big concern IMO.